include/openthread/instance.h

```diff
@@ -114,6 +114,15 @@ otInstance *otInstanceInitSingle(void);
  */
 otInstance *otInstanceInitMultiple(uint8_t aIdx);
 
+/**
```

Comment:
Since this is updating the public OT APIs, need to also increment the `OPENTHREAD_API_VERSION` in this header.